msg/Status.msg

```diff
@@ -1,9 +1,21 @@
 std_msgs/Header header
 
-uint8 status
-uint32 particle_size
-float32 match_ratio
+# status definitions
 
 uint8 NORMAL = 0
 uint8 GLOBAL_LOCALIZATION = 1
 uint8 EXPANSION_RESETTING = 2
+uint8 COVARIANCE_CONVERGED = 3
+
+uint8 status
+
+# error definitions
+
+uint8 GOING_WELL = 0
+uint8 LARGE_STD_VALUE = 1
```

Comment:
I would like not to mix system status/error and estimation result things (`COVARIANCE_CONVERGED` and `LARGE_STD_VALUE`).
Adding `convergence_status` would be better.